1-exercises/B-while-loop/exercise.js

```diff
@@ -7,6 +7,18 @@
 
 function evenNumbers(n) {
     // TODO
+    i = 0;
+    y = n;
+    let x = "";
+    if (n > 0) {
+      while (i < y) {
+        x += i + ",";
+        i += 2;
+      }
+      console.log(x);
+    } else {
+      //   console.log("nothing");
+    }
```

Comment:
Nice try, but it doesn't cover all the use cases.
Hint: if you create an array where you stock all the first n even values, using the length of that value as a stopping condition for your while loop, it will be easier to display then